lib/elixir/lib/enum.ex

```diff
@@ -249,7 +249,10 @@ defmodule Enum do
   @type t :: Enumerable.t()
   @type acc :: any
   @type element :: any
+
+  @typedoc "Zero-based index. It can also be negative integer."
```

Comment:
"It can also be **a** negative integer."?

Reply:
done